web_src/js/features/repo-migrate.js

```diff
@@ -53,3 +56,14 @@ export function initRepoMigrationStatusChecker() {
 
   syncTaskStatus(); // no await
 }
+
+async function doRetry(e) {
```

Comment:
```suggestion
async function doRetry(e) {
```
`doRetry` can you rename to doMigrationRetry or something meaningful.. which will be easy to understand and maintain. `doRetry ` is too common.